lib/formatter.js

```diff
@@ -23,18 +23,15 @@ export default {
   // formats the given text editor
   formatTextEditor(editor, range = null) {
     try {
-      const { status, stdout, stderr } = this.runFormat(
+      const { status, stdout, stderr, error } = this.runFormat(
         this.getTextInRange(editor, range)
       );
 
-      switch (status) {
-      case 0: {
+      if (status == 0 && !stderr.length && !error) {
         this.insertText(editor, range, stdout.toString());
-        break;
-      }
-      default:
+      } else {
         this.showErrorNotifcation("Elixir Formatter Error", {
-          detail: stderr
+          detail: stderr || error.message
```

Comment:
Good news – the `detail` arg will have `toString()` called on it automatically, so this can be `detail: stderr || error`. This is nice since we also don't have to null check `error` before calling `message`.